pod/perlop.pod

```diff
@@ -2875,85 +2877,153 @@ must use an C<eval()>:
 
     eval "tr/$oldlist/$newlist/, 1" or die $@;
 
-=item C<< <<I<EOF> >>
-X<here-doc> X<heredoc> X<here-document> X<<< << >>>
+=head3 Here-docs
+X<here-doc> X<here-docs> X<heredoc> X<here-document> X<<< << >>>
+
+=over 4
+
+=item C<< <<I<EOT> >>
+
+=item C<< <<~I<EOT> >>
+
+=back
```

Comment:
I think this should also include the following paragraph, or alternatively since this is a whole section on here-docs, the item headings might be superfluous now.

Reply:
I removed them; see what you think now

Reply:
I was referring to the headings such as `=item C<< <<I<EOT> >>`. It could still be useful to have the other subheadings you added.

Reply:
How about now?